pkg/sync/rpc/list.go

```diff
@@ -90,32 +111,34 @@ func (*BackupList) remove(b []*BackupInfo, index int) ([]*BackupInfo, error) {
 }
 
 // Refresh deletes all the old completed backups from the front. Old backups are the completed backups
-// that are created before MaxBackupSize completed backups
+// that are created before the number we choose to retain.
 func (b *BackupList) refresh() error {
 	b.Lock()
 	defer b.Unlock()
 
-	var index, completed int
+	for state, limit := range retainBackupStateCounts {
+		var index, completed int
```

Comment:
The variable `completed` is a bit confusing. It used to count both `error` and `completed` backups. Should we change it to `count` instead?